INFO     common.logger:test_demo_api.py:114 Test under execution : test_004
INFO     common.logger:rest_client.py:95 Initializing POST rest call from https://api.postman.com/workspaces?
INFO     common.logger:rest_client.py:121 POST request is Successful.....
INFO     common.logger:test_demo_api.py:150 Initial workspace name: created_from_python_framework
INFO     common.logger:test_demo_api.py:151 Initial workspace id : d8db8892-405a-42b3-a8d5-3a9cb5989190
INFO     common.logger:user_asserts.py:13 Actual and expected should match
INFO     common.logger:user_asserts.py:14 Expected value - 200 : Actual value - 200
INFO     common.logger:user_asserts.py:13 None
INFO     common.logger:user_asserts.py:14 Expected value - created_from_python_framework : Actual value - created_from_python_framework
INFO     common.logger:rest_client.py:95 Initializing PUT rest call from https://api.postman.com/workspaces/d8db8892-405a-42b3-a8d5-3a9cb5989190
INFO     common.logger:rest_client.py:121 PUT request is Successful.....
INFO     common.logger:test_demo_api.py:186 Workspace name after put : new_name
INFO     common.logger:test_demo_api.py:187 Workspace id after put: d8db8892-405a-42b3-a8d5-3a9cb5989190
INFO     common.logger:user_asserts.py:13 Actual and expected should match
INFO     common.logger:user_asserts.py:14 Expected value - 200 : Actual value - 200
INFO     common.logger:user_asserts.py:19 None
INFO     common.logger:user_asserts.py:20 Expected value - created_from_python_framework : Actual value - new_name